homeassistant/components/water_heater/strings.json

```
@@ -4,5 +4,15 @@
       "turn_on": "Turn on {entity_name}",
       "turn_off": "Turn off {entity_name}"
     }
+  },
+  "state": {
+    "_": {
+      "off": "[%key:common::state::off%]",
+      "eco": "Eco",
+      "electric": "Electric",
```

You're missing performance.

```suggestion
      "electric": "Electric",
      "performance": "Performance",
```

---

Oops, good catch. Fixed in a0c725e.